client/components/public/FAQ.jsx

```diff
@@ -515,3 +510,5 @@ FAQ = class FAQ extends Component {
     );
   }
 }
+
+FAQ = GamestateComp(FAQ);
```

Comment:
no need to do it now, but if you do append to a file, can you include a newline at the end? :)

Reply:
Done!